frontend/src/components/algolia/algoliaInstantSearch.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import algoliasearch from 'algoliasearch/lite';
+import {
+  Highlight, Hits, InstantSearch,
+
+
+  Pagination, SearchBox
+} from 'react-instantsearch-dom';
+import PropTypes from 'prop-types';
+import '../../styles/App.css';
+import styled from "styled-components";
+import { css } from "styled-components/macro"; //eslint-disable-line
+
+
+
+
+
+const searchClient = algoliasearch(
+  'A0IP7ZFQR8',
+  '7d853e1791e8f7668cb942eeb76888ac'
+);
```

Comment:
Of course we should not commit API keys into repo, even test ones. Move this into the local (ignored) env file and use variables here.

Reply:
created .env.local file and this works locally. 

## NOTE from ```create-react-app``` docs: 

> WARNING: Do not store any secrets (such as private API keys) in your React app!

> Environment variables are embedded into the build, meaning anyone can view them by inspecting your app's files.

https://create-react-app.dev/docs/adding-custom-environment-variables/

After reading this, I am not sure if I should push this file? Since it is ignored it will not be pushed. But the docs say that after build these values will be exposed. This is why Firebase probably has a place to enter thise values.

Reply:
You need not pushing this file. The proper values of these variables will be provided to React in our deployment pipeline, so they will be embedded, yes. That's not a problem. But we should not commit them into the repository.

Reply:
If I don't push the file in this section will not be visible. I have inserted the environment variable and it works as expected. 

do you mean I should not push the local environment variable file?￼

Reply:
Yes I mean not to push the local env file. However, please describe needed variables in the frontend's README file. Similar to this: https://github.com/PureBhaktiArchive/audioseva#frontend-environment-variables